src/i965_drv_video.c

```diff
@@ -6408,6 +6408,25 @@ i965_QuerySurfaceAttributes(VADriverContextP ctx,
     attribs[i].value.value.i = max_height;
     i++;
 
+    if (obj_config->entrypoint == VAEntrypointEncSlice ||
+        obj_config->entrypoint == VAEntrypointEncSliceLP) {
+        attribs[i].type = VASurfaceAttribMinWidth;
+        attribs[i].value.type = VAGenericValueTypeInteger;
+        attribs[i].flags = VA_SURFACE_ATTRIB_GETTABLE;
+        attribs[i].value.value.i = 32;
+        if (obj_config->profile == VAProfileJPEGBaseline)
```

Comment:
VAProfileJPEGBaseline only has VAEntrypointEncPicture encode entrypoint.  Thus, this will never be true.